patterns/homepage.php

```diff
@@ -0,0 +1,230 @@
+<figure class="wp-block-image size-full is-style-course-image-border" style="border-radius:8px"><img src="http://sentest.test/wp-content/themes/sensei-theme/assets/images/course-theme-testimonial.png" alt="" class="wp-image-2150"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"bottom","style":{"spacing":{"blockGap":"10px"}}} -->
+<div class="wp-block-column is-vertically-aligned-bottom"><!-- wp:heading {"style":{"spacing":{"margin":{"bottom":"30px"}},"typography":{"textTransform":"uppercase"}},"textColor":"tertiary","fontSize":"x-large"} -->
+<h2 class="has-tertiary-color has-text-color has-x-large-font-size" style="text-transform:uppercase;margin-bottom:30px">“I always wanted to write, and<br>thanks to Cours, I got it right. My<br>writing is clearer, and I can finally get my message across.”</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"500"}},"className":"has-system-font-family"} -->
+<p class="has-system-font-family" style="font-style:normal;font-weight:500">Christopher Brown</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"className":"has-system-font-family","fontSize":"xx-small"} -->
+<p class="has-system-font-family has-xx-small-font-size">Founder at BeautifulWriting.com</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"40px"} -->
+<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}}},"backgroundColor":"background","textColor":"primary","className":"course-home-page-responsive-padding","layout":{"type":"default","contentSize":"1200px"}} -->
+<div class="wp-block-group alignfull course-home-page-responsive-padding has-primary-color has-background-background-color has-text-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:spacer {"height":"10px"} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"primary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-primary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-gothic-font-family"} -->
+<h3 class="has-gothic-font-family" style="text-transform:uppercase">pages</h3>
+<!-- /wp:heading -->
+
+<!-- wp:group {"className":"has-eb-garamond-font-family","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group has-eb-garamond-font-family"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size">About</p>
```

Comment:
I think these links are actually page links, should we use a nav block?

Reply:
Thanks @gabrielcaires , I looked a bit into using nav block here. It seems nav block is more suitable for the navbar and it's mostly used in that context. I think maybe we can keep it this way for now